pdfarranger/exporter.py

```diff
@@ -75,7 +75,7 @@ def _intersect_rectangle(rect1, rect2):
     ]
 
 
-def _mediabox(page, crop):
+def _mediabox(page, crop=[0., 0., 0., 0.]):
```

Comment:
Good catch by codacy: This is dangerous, and one of the worst pitfalls in Python imo. While it may work with the current implementation it may break horribly when the code is changed to modify this reference to the "crop" singleton, it can lead to bugs which are horribly hard to debug.

Just make the default value None or maybe 0 in this case and add an if to replace it with the list.

Reply:
This is fixed